src/main/java/org/testng/internal/ConfigurationMethod.java

```diff
@@ -455,5 +455,14 @@ public boolean isLastTimeOnly() {
     return result;
   }
 
+  public String[] getGroupFilters() {
```

Comment:
Again a detail and it's just a personal taste but I prefer to see the business code at the root level of the method
```java
   IBeforeMethod before = m_annotationFinder.findAnnotation(getConstructorOrMethod(), IBeforeMethod.class);
   if (before == null) {
     return new String[0];
   }
   return before.getGroupFilters();
```